pf9_saml_auth/v3/base.py

```diff
@@ -15,7 +15,7 @@
 """BasePF9SAMLPlugin class."""
 
 import base64
-import urlparse
+from six.moves import urllib
```

Comment:
nit: from six.moves.urllib import parse

and change line80 to "parse.urlparse"

Reply:
will change that.